stdlib/Distributed/src/managers.jl

```diff
@@ -394,7 +394,11 @@ function connect(manager::ClusterManager, pid::Int, config::WorkerConfig)
 
     # master connecting to workers
     if config.io !== nothing
-        (bind_addr, port) = read_worker_host_port(config.io)
+        try
+            (bind_addr, port) = read_worker_host_port(config.io)
+        catch e
+            error(e.msg)
```

Comment:
What's the reason for this? Looks unnecessary to me, plus could cause a problem if `e` doesn't have a `msg` field.

Reply:
emitting the error msg is now handled by a `finally` block with a `rethrow` in `create_worker()`